### PR TITLE
[CPDE-1423] Caching in Github pipelines node_modules

### PR DESCRIPTION
**Pull request by andreyvalynko-int**

### CODE REVIEW

File: .github/workflows/publish-apispec.yaml [hunk 1]
```diff
@@ -32,10 +32,20 @@ jobs:
       - name: Check out repository code
         uses: actions/checkout@v3
 
+      - name: Cache node modules
```
Comment (arnikasky) [inline .github/workflows/publish-apispec.yaml]: Please, remove this

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -36,7 +36,17 @@ jobs:
       - name: 'Workaround permissions issue'
         run: chown $(whoami):$(whoami) -R .
 
+      - name: Cache node modules
```
Comment (arnikasky) [inline .github/workflows/release.yaml]: Please, remove this

File: .github/workflows/semantic-release-v2.yaml [hunk 3]
```diff
@@ -56,7 +56,17 @@ jobs:
       - name: 'Workaround permissions issue'
         run: chown $(whoami):$(whoami) -R .
 
+      - name: Cache node modules
```
Comment (arnikasky) [inline .github/workflows/semantic-release-v2.yaml]: Please, remove this

File: .github/workflows/semantic-release.yaml [hunk 4]
```diff
@@ -52,7 +52,17 @@ jobs:
       - name: 'Workaround permissions issue'
         run: chown $(whoami):$(whoami) -R .
 
+      - name: Cache node modules
```
Comment (arnikasky) [inline .github/workflows/semantic-release.yaml]: Please, remove this

**Review by arnikasky — CHANGES_REQUESTED** [review]

**Review by arnikasky — APPROVED** [review]

**Review by euqen — APPROVED** [review]

**Pull request merged**